test/routers/personalCodeRouter.int.ts

```diff
@@ -40,28 +35,8 @@ describe("personal code router tests", () => {
         expect(middlewareMocks.mockSessionMiddleware).toHaveBeenCalledTimes(1);
     });
 
-    it("Should render the Personal Code page with a success status code and correct links", async () => {
-        const queryParams = new URLSearchParams("lang=en");
-        const uriWithQuery = `${PrefixedUrls.PERSONAL_CODE}?${queryParams}`;
-        const uri = getUrlWithTransactionIdAndSubmissionId(uriWithQuery, TRANSACTION_ID, PSC_VERIFICATION_ID);
-
-        const resp = await request(app).get(uri);
-
-        const $ = cheerio.load(resp.text);
-
-        expect(resp.status).toBe(HttpStatusCode.Ok);
-        expect($("a.govuk-back-link").attr("href")).toBe("/persons-with-significant-control-verification/transaction/11111-22222-33333/submission/662a0de6a2c6f9aead0f32ab/individual/psc-list?lang=en");
-    });
-
-    it("Should redirect to Individual Statement screen", async () => {
-        const expectedRedirectUrl = `${PrefixedUrls.INDIVIDUAL_STATEMENT.replace(":transactionId", TRANSACTION_ID).replace(":submissionId", PSC_VERIFICATION_ID)}?lang=en`;
-        await request(app)
-            .post(getUrlWithTransactionIdAndSubmissionId(PrefixedUrls.PERSONAL_CODE, TRANSACTION_ID, PSC_VERIFICATION_ID))
-            .send({ pscType: "individual" })
-            .set({ "Content-Type": "application/json" })
-            .query({ companyNumber: COMPANY_NUMBER, lang: "en", pscType: "individual" })
-            .expect(HttpStatusCode.Found)
-            .expect("Location", expectedRedirectUrl);
+    it("Should render the personal code page with a successful status code", async () => {
+        const resp = await request(app).get(PrefixedUrls.PERSONAL_CODE);
+        expect(resp.status).toBe(200);
```

Comment:
ought there be a `(app).post` test in here too?  I know that this is done in the new handler tests, but I guess to be consistent it ought to be done here too.  pscTypeRouter has one.

Reply:
following dev discussion 11/6 we will wait for the routers and handlers to be refactored, and part of that will be re-organising the tests.  So my comment can be covered there.